tests/php/ContentReviewReportTest.php

```diff
@@ -52,22 +52,23 @@ public function testPagesDueForReviewReport()
             "ReviewDateBefore" => "2010-12-12",
         ]);
 
-        $this->assertListEquals([
-            ['Title' => 'Contact Us Child'],
-            ['Title' => 'Home'],
-            ['Title' => 'About Us'],
-            ['Title' => 'Staff'],
-            ['Title' => 'Contact Us'],
-        ], $results);
+        $recordTitles = $results->column('Title');
+        $this->assertContains('Contact Us Child', $recordTitles);
+        $this->assertContains('Home', $recordTitles);
+        $this->assertContains('About Us', $recordTitles);
+        $this->assertContains('Staff', $recordTitles);
+        $this->assertContains('Contact Us', $recordTitles);
```

Comment:
`assertCount(5, $recordTitles)` might be a good idea here so that we don't have other records we're not expecting. Same for elsewhere in the PR

Reply:
Sure